cmake/config/template-arguments.in

```diff
@@ -69,20 +69,18 @@ REAL_SERIAL_VECTORS := { Vector<double>;
                     @DEAL_II_EXPAND_PETSC_MPI_BLOCKVECTOR_REAL@;
                   }
 
-REAL_NONBLOCK_VECTORS := { Vector<double>;
-                    Vector<float> ;
-
-                    LinearAlgebra::Vector<double>;
-                    LinearAlgebra::Vector<float> ;
-
+REAL_NONBLOCK_VECTORS_MF := { 
                     LinearAlgebra::distributed::Vector<double>;
                     LinearAlgebra::distributed::Vector<float> ;
-
-                    @DEAL_II_EXPAND_TRILINOS_VECTOR@;
+                  }
+                  
+REAL_NONBLOCK_VECTORS := { 
```

Comment:
this one should cover 90% of usage when projecting from `FEValues`-kind of quadrature data (pair of cell iterator and quadrature point). Mostly with MPI but I tried to keep a few vectors in case someone does serial calculation with deal.ii's matrices.